pandas/core/arrays/masked.py

```diff
@@ -273,4 +273,4 @@ def value_counts(self, dropna: bool = True) -> "Series":
         mask = np.zeros(len(counts), dtype="bool")
         counts = IntegerArray(counts, mask)
 
-        return Series(counts, index=index)
+        return Series(counts, index=index, dtype="Int64")
```

Comment:
is there a reason we are forcing this to Int64, I think it will be anyhow (as that's the default)